notebooks/advanced-feature-transformations-explain-local.ipynb

```diff
@@ -351,7 +352,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true
```

Comment:
remove

Reply:
I removed this but am concerned about how it got there in the first place? talk on mon?